rust/src/mqtt/mqtt.rs

```diff
@@ -830,6 +828,13 @@ pub unsafe extern "C" fn rs_mqtt_register_parser(cfg_max_msg_len: u32) {
                 SCLogError!("Invalid value for mqtt.max-tx");
             }
         }
+        if let Some(val) = conf_get("app-layer.protocols.mqtt.max-msg-length") {
+            if let Ok(v) = get_memval(val) {
+                MAX_MSG_LEN = v as u32;
+            } else {
+                SCLogError!("Invalid value for mqtt.max-msg-length: {}", val);
```

Comment:
not new in this PR, but this is an error message acting like a warning. I think we have several more of these, this deserves a ticket I think.

Reply:
[Issue 6652](https://redmine.openinfosecfoundation.org/issues/6652)